pkg/tuned/dialog.js

```diff
@@ -104,14 +104,12 @@ function setup() {
 
                     button.text(state == "running" ? active : _("none"));
                     button.prop('disabled', state == "not-installed" || !permission.allowed);
-                    button.toggleClass('disabled', state == "not-installed" || !permission.allowed);
```

Comment:
Should be safe to remove?

Reply:
Probably. I would look at the page and see if it behaves as expected without this one line.

Reply:
Can you point me where this is?

Reply:
It works without this.
![desktop](https://user-images.githubusercontent.com/12330670/74930896-d505e580-53de-11ea-9707-1f59bcf4df25.png)
It is that `desktop` text.

Reply:
👍